runfo/GetFromHelixOptionSet.cs

```diff
@@ -12,10 +12,13 @@ internal sealed class GetFromHelixOptionSet : OptionSet
 
         internal string? DownloadDir { get; set; }
 
+        internal string? Token { get; set; }
+
         internal GetFromHelixOptionSet()
         {
             Add("j|jobid=", "helix job id to download items from.", j => JobId = j);
             Add("o|output=", "output directory to download to.", d => DownloadDir = d);
+            Add("t|token=", "Helix authentication token in order to get payload from an internal job.", t => Token = t);
```

Comment:
won't this conflict with the `-token` argument for specifying the AzDO token?

Reply:
That's true. Good catch, I will change the name to something like `helixtoken`.